Gemfile

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'bcrypt', '3.1.12'
+gem 'bootstrap-sass', '3.3.7'
+gem 'coffee-rails', '4.2.2'
+gem 'dotenv-rails'
+gem 'devise'
+gem 'jbuilder', '2.7.0'
+gem 'jquery-rails', '4.3.1'
+gem 'omniauth-facebook'
+gem 'puma', '3.9.1'
+gem 'rails', '5.1.6'
+gem 'sass-rails', '5.0.6'
+gem 'turbolinks', '5.0.1'
+gem 'uglifier', '3.2.0'
+
+group :development, :test do
+  gem 'byebug', '9.0.6', platform: :mri
+  gem 'rspec-rails'
+  gem 'sqlite3', '1.3.13'
+end
+
+group :development do
+  gem 'capybara', '2.7.1'
+  gem 'listen', '3.1.5'
+  gem 'spring', '2.0.2'
+  gem 'spring-watcher-listen', '2.0.1'
+  gem 'web-console', '3.5.1'
+end
+
+group :test do
+  gem 'guard', '2.13.0'
+  gem 'guard-minitest', '2.4.4'
+  gem 'minitest', '5.10.3'
+  gem 'minitest-reporters', '1.1.14'
+  gem 'rails-controller-testing', '1.0.2'
+end
+
+group :production do
+  gem 'pg', '0.20.0'
```

Comment:
`pg` should be used in place of sqlite3 not only for production but also development.

Reply:
The last commit wasn't pushed correctly so the gemfile didnt update, should be good now, apologies :)